nvflare/fuel/utils/log_utils.py

```diff
@@ -31,3 +101,45 @@ def add_log_file_handler(log_file_name):
     file_handler.setLevel(main_handler.level)
     file_handler.setFormatter(main_handler.formatter)
     root_logger.addHandler(file_handler)
+
+
+def print_logger_hierarchy(package_name="nvflare", level_colors=DEFAULT_LEVEL_COLORS):
```

Comment:
later we could change this to "format_logger_hierarchy" that returns a string or json and can be used to print or pass it to the user (for admin commands)